upgrade/sql/8.0.0.sql

```diff
@@ -192,6 +192,8 @@ ALTER TABLE `PREFIX_product_shop` MODIFY COLUMN `redirect_type` ENUM(
     '404', '410', '301-product', '302-product', '301-category', '302-category'
 ) NOT NULL DEFAULT '404';
 
+ALTER TABLE `PREFIX_tab` ADD route_name VARCHAR(256) DEFAULT NULL;
```

Comment:
oh 🤯

Reply:
I actually try modifying the autoupgrade to run first the doctrine migration, but it doesn't work, it stop after the doctrine migration and never try to run the .sql part (I didn't have time to investigate why it's working like this)

Reply:
That's weird, I don't understand why you need to handle this manually if Doctrine already does it as you said 🤔 Wat is this import you are referring to?

Besides if we do need to force this manually why not I don't mind, however, this field was introduced in 1.7.7 https://github.com/PrestaShop/autoupgrade/pull/571/files so you'd need to do this in 1.7.7.0.sql file I guess

Reply:
Oh by import you mean the tabs that are inserted in the `ps_800_add_security_tab` function maybe? I understand why this would fail then if doctrine scripts are executed after this SQL and PHP function, and it makes sense that you change this here right before the method is called But maybe out of safety this should also be done in 1.7.7.0.sql

In the future we'll need to get rid of the usage of this doctrine command, it shouldn't be used in production but only to help us build these SQL files. We'll need to go back to previous versions and run the command to integrate the changes back into these files (version by version) so we can then remove the usage of the command completely during the upgrade

Reply:
@jolelievre I agree for the doctrine commands, it should be either full doctrine, or not at all, having a mix is not a good idea an actually the doctrine structure could be at the end different of what we have in the .sql
By import I mean the execution of the 8.0.0.sql file, which executes ps_800_add_security_tab function and needs this field. So the new field should have been definitely initially in the 1.7.7.0.sql file.